src/bgw/job_stat.c

```diff
@@ -19,6 +19,7 @@
 #include <utils/fmgrprotos.h>
 #endif
 #define MAX_INTERVALS_BACKOFF 5
+#define MAX_FAILURES_MULTIPLIER 20
```

Comment:
I would prefer to have a MAX_INTERVAL instead. Then we could use interval_div(max, multiplier) and if that answer is less than the original interval, use the max_interval instead.

Reply:
@cevian the suggestion is not clear to me. Are you saying limit next start by MAX_INTERVAL instead? Could you elaborate?